mocks/manager.go

```diff
@@ -1,5 +1,10 @@
 // Code generated by MockGen. DO NOT EDIT.
 // Source: github.com/Layr-Labs/eigenda-proxy/store (interfaces: IManager)
+//
+// Generated by this command:
+//
+//	mockgen -package mocks --destination ../mocks/manager.go . IManager
+//
```

Comment:
There's also `make go-gen-mocks`

Reply:
I did use `make go-gen-mocks`. Not sure why this repo was setup to use the deprecated go mock library instead of the new uber mock repo. Updated that dependency in go.mod.

Created this issue to make a reproducible dev env: https://github.com/Layr-Labs/eigenda-proxy/issues/236